login.html

```diff
@@ -4,8 +4,44 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>판다마켓</title>
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/basic.css" />
+    <link rel="stylesheet" href="css/form.css" />
+    <link
+      rel="stylesheet"
+      as="style"
+      crossorigin
+      href="https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/static/pretendard.min.css"
+    />
   </head>
   <body>
-    LOGIN
+    <div class="form_container">
+      <a href="index.html">
+        <img
+          class="form_logo_img"
+          src="img/login_logo.png"
+          alt="로그인페이지_판다마켓_로고_이미지"
+        />
+      </a>
+      <p class="form_text_input">이메일</p>
+      <input type="text" class="form_input" />
```

Comment:
❗️ 수정요청
특정 input이 어떤 값을 받는지 설명하는 text들을 label 이라고 합니다.
이러한 친구들은 일반적으로 마우스로 선택시 연결된 input으로 focus가 이동하는 기능이 있습니다~
이를 위해, 그리고 명확한 연결관계 표시하기 위해 label 태그를 사용해주시는 것이 좋습니다~

```suggestion
      <label class="form_text_input" for="email">이메일</label>
      <input id="email" type="text" class="form_input" />
```